### PR TITLE
Scripts cleanup

### PR DESCRIPTION
**Pull request by garrett**
~This needs to be rebased once #426 lands.~

### CODE REVIEW

File: _scripts/toolbox-run [hunk 1]
```diff
@@ -1,3 +1,8 @@
-#!/bin/sh
+#!/bin/bash
 
-toolbox run -c cockpit-website bundle exec jekyll server $@
+if [[ -n $1 ]]
```
Comment (martinpitt) [inline _scripts/toolbox-run:3]: To me this is a bit unfamiliar and a bashism. The canonical/POSIX way is `if [ -n "$1" ]`. But should be ok.

**Review by martinpitt — APPROVED** [review]
Thanks! This is still draft, but I think approvals survive undrafting.

File: _scripts/toolbox-run (hunk 1 above)
Comment (garrett) [reply _scripts/toolbox-run:3]: It's definitely a bashism: https://stackoverflow.com/questions/3427872/whats-the-difference-between-and-in-bash

I could change it to what you suggest and switch back to sh though; probably fine here.

**Comment by garrett** [comment]
Not quite done with this PR, but it's getting close.

**Pull request merged**